docs/using_lagoon/docker_images/rabbitmq.md

```diff
@@ -0,0 +1,34 @@
+# rabbitmq Image
+amazee.io RabbitMQ Dockerfile with management plugin installed, based on the official rabbitmq:3-management image at [docker-hub](https://hub.docker.com/_/rabbitmq).
```

Comment:
We're not using "amazee.io" everything is called "Lagoon", please rename

Reply:
Ok. Should I change also nxing and php readme.md pages?

Reply:
yes if you want, but in a separate PR

Reply:
Ok